experiment/measurer/measure_manager.py

```diff
@@ -755,51 +700,294 @@ def get_pool_args(measurers_cpus, runners_cpus):
+        measurements tasks from workers, retrieve measurement results from
+        response queue and writes measured snapshots in database."""
+        logger.info('Starting measure manager loop.')
+        with multiprocessing.Pool() as pool:
+            set_up_coverage_binaries(pool, self.experiment)
+        (request_queue, response_queue) = self.initialize_queues()
+        self.start_workers(request_queue, response_queue)
         max_cycle = _time_to_cycle(max_total_time)
         queued_snapshots = set()
-        while not scheduler.all_trials_ended(experiment):
-            continue_inner_loop = measure_manager_inner_loop(
-                experiment, max_cycle, request_queue, response_queue,
-                queued_snapshots)
+        while not scheduler.all_trials_ended(self.experiment):
+            continue_inner_loop = self.measure_manager_inner_loop(
+                max_cycle, request_queue, response_queue, queued_snapshots)
             if not continue_inner_loop:
                 break
             time.sleep(MEASUREMENT_LOOP_WAIT)
         logger.info('All trials ended. Ending measure manager loop')
 
 
+class LocalMeasureManager(BaseMeasureManager):
+    """Class that holds implementations of core methods for running a measure
+    worker locally."""
+
+    def __init__(self,
+                 experiment: str,
+                 region_coverage=False,
+                 measurers_cpus=None):
+        super().__init__(experiment, region_coverage)
+        self.measurers_cpus = measurers_cpus
+
+    def initialize_queues(
+        self
+    ) -> Tuple[multiprocessing.queues.Queue, multiprocessing.queues.Queue]:
+        return (multiprocessing.Queue(), multiprocessing.Queue())
+
+    def start_workers(self, request_queue: multiprocessing.queues.Queue,
+                      response_queue: multiprocessing.queues.Queue):
+        if not self.measurers_cpus:
+            self.measurers_cpus = multiprocessing.cpu_count()
+            logger.info(
+                'Number of measurer CPUs not passed as argument. using %d',
+                self.measurers_cpus)
+        with multiprocessing.Pool(processes=self.measurers_cpus) as pool:
+            config = {
+                'request_queue': request_queue,
+                'response_queue': response_queue,
+                'region_coverage': self.region_coverage,
+            }
+            local_measure_worker = measure_worker.LocalMeasureWorker(config)
+
+            # Since each worker is going to be in an infinite loop, we dont need
+            # result return. Workers' life scope will end automatically when
+            # there are no more snapshots left to measure.
+            logger.info('Starting measure worker loop for %d workers',
+                        self.measurers_cpus)
+            for _ in range(self.measurers_cpus):
+                _result = pool.apply_async(
+                    local_measure_worker.measure_worker_loop)
```

Comment:
Why we do need to store the return value to `_result`?

Reply:
We actually don't need it, I will remove that! thx